src/main/resources/application.yml

```diff
@@ -8,13 +8,21 @@ spring:
       discovery:
         enabled: true
         service-id: config
-    
+
+    consul:
+      host: localhost
+      port: 8500
+      discovery:
+        prefer-ip-address: true
     
     gateway:
+      # CORS shouldn't be an issue in the dev/staging environments,
+      # but this is here so a local instance of the frontend can
+      # connect to the dev cluster
       globalcors:
         corsConfigurations:
           '[/**]':
-            allowedOrigins: "http://localhost:4200"
+            allowedOrigins: "localhost:4200"
```

Comment:
CORS rules require the origins to include the scheme (http://). if you wanted to include https (not relevant for a local ng serve), that's be a separate origin.